content/docs/guides/logical-replication-decodable.md

```diff
@@ -0,0 +1,194 @@
+ALTER DEFAULT PRIVILEGES IN SCHEMA <schema_name> GRANT SELECT ON TABLES TO <role_name>;
+```
+
+Granting `SELECT ON ALL TABLES IN SCHEMA` instead of naming the specific tables avoids having to add privileges later if you add tables to your publication.
+
+## Create a publication
+
+For each table you would like to ingest into Decodable, set its [replica identity](https://www.postgresql.org/docs/current/logical-replication-publication.html) to `FULL`.
+To do so, issue the following statement in the **Neon SQL Editor**:
+
+```sql
+ALTER TABLE <tbl1> REPLICA IDENTITY FULL;
+```
+
+Next, create a [publication](https://www.postgresql.org/docs/current/sql-createpublication.html) with the name `dbz_publication`. Include all the tables you would like to ingest into Decodable.
+
+```sql
+CREATE PUBLICATION dbz_publication FOR TABLE <tbl1, tbl2, tbl3>;
+```
+
+Refer to the [Postgres docs](https://www.postgresql.org/docs/current/sql-alterpublication.html) if you need to add or remove tables from your publication.
+Alternatively, you also can create a publication `FOR ALL TABLES`.
+
+Upon start-up, the Decodable connector for Postgres will automatically create the [replication slot](https://www.postgresql.org/docs/current/logicaldecoding-explanation.html#LOGICALDECODING-REPLICATION-SLOTS) required for ingesting data change events from Postgres.
+The slot's name will be prefixed with `decodable_`, followed by a unique identifier.
+
+<Admonition type="important">
+To prevent storage bloat, **Neon automatically removes _inactive_ replication slots after a period of time if there are other _active_ replication slots**. If you have or intend on having more than one replication slot, please see [Unused replication slots](/docs/guides/logical-replication-neon#unused-replication-slots) to learn more.
+</Admonition>
+
+## Allow inbound traffic
+
+If you are using Neon's **IP Allow** feature to limit the IP addresses that can connect to Neon, you will need to allow inbound traffic from Decodable's IP addresses. 
+Refer to the [Decodable documentation](https://docs.decodable.co/reference/regions-and-ip-addresses.html#ip-addresses) for the list of IPs that need to be allowlisted for the Decodable region of your account.
+For information about configuring allowed IPs in Neon, see [Configure IP Allow](/docs/manage/projects#configure-ip-allow).
+
+## Create a Postgres source connector in Decodable
+
+1. In the Decodable web UI, select **Connections** from the left navigation bar and click "New Connection".
+2. In the connector catalog, choose "Postgres CDC" and click "Connect".
+3. Enter the connection details for your Neon database. You can get these details from your Neon connection string, which you'll find in the **Connection Details** widget on the **Dashboard** of your Neon project.
+   Your connection string will look like this:
+
+   ```bash shouldWrap
+   postgres://alex:AbC123dEf@ep-cool-darkness-123456.us-east-2.aws.neon.tech/dbname?sslmode=require
```

Comment:
do the username/password/hostname here have special meanings? Because I'd be inclined to just use the more generic option like this:

```suggestion
   postgres://user:password@fqdn.my.neon.instance.neon.tech/dbname?sslmode=require
```

Reply:
I copied this from other existing guides. I'll leave it as is, for the sake of consistency.